src/main/java/io/codelex/TESTS/Joiner.java

```diff
@@ -0,0 +1,21 @@
+package io.codelex.TESTS;
+
+public class Joiner<Object> {
```

Comment:
Te ir jāizmanto T nevis Object, arī uzdevumā bija konkrēti pateikts: `Design the Joiner<T> ...`